xblockmufi/settings.py

```diff
@@ -0,0 +1,39 @@
+"""
+Settings for mufi xblock
+"""
+import os
+
+
+DATABASES = {
+    'default': {
+        'ENGINE': 'django.db.backends.sqlite3',
+        # 'NAME': 'intentionally-omitted',
+    },
+}
+
+TEST_RUNNER = 'django_nose.NoseTestSuiteRunner'
+
+INSTALLED_APPS = (
+    'django_nose',
+    'xblockmufi',
+)
+
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+TEMPLATES = [
```

Comment:
Do we need this part? Do other XBlocks have it?
I don't remember seeing it in the free-text-xblock...

Reply:
I don't think the other XBlocks have it. So just delete the TEMPLATES field?